components/FormStyles/RadioButton.tsx

```diff
@@ -31,3 +29,33 @@ export const RadioButton = styled.input`
     color: white;
   }
 `
+
+export const RadioInputIcon = styled.span`
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  box-shadow: inset 0 0 0 1px rgba(16, 22, 26, 0.2),
+    inset 0 -1px 0 rgba(16, 22, 26, 0.1);
+  background-color: #f5f8fa;
+  background-image: linear-gradient(
```

Comment:
what does this linear-gradient do

Reply:
makes it pretty (short answer I have no idea but it makes it work nice, I stole it from the internet)